src/styles/blocks/product.scss

```
@@ -0,0 +1,96 @@
+.product {
+  width: 200px;
+  padding: 16px;
+  margin-right: 48px;
+  margin-bottom: 46px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 16px auto 40px;
+  }
+
+  &__name {
+    margin-bottom: 4px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $text-color-dark;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    color: $text-color-light;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+
+    &-feedback {
+      display: flex;
+      align-items: center;
+      font-size: 10px;
+      font-weight: 400;
+      color: $text-color-dark;
+    }
+
+  }
```

Rewatch video about naming in BEM [video](https://mate.academy/learn/css-advanced/bem#/video/920)

rating-feedback isn't a modifier. It's an element name.
```suggestion
  &__rating {
    display: flex;
    justify-content: space-between;
  }
  &raiting-feedback {
      display: flex;
      align-items: center;
      font-size: 10px;
      font-weight: 400;
      color: $text-color-dark;
    }
```